docs/topics/interfaces.md

```diff
@@ -108,10 +107,10 @@ class D : A, B {
 }
 ```
 
-Interfaces *A* and *B* both declare functions *foo()* and *bar()*. Both of them implement *foo()*, but only *B* implements 
-*bar()* (*bar()* is not marked abstract in *A*, because this is the default for interfaces, if the function has no body). 
+Interfaces *A* and *B* both declare functions *foo()* and *bar()*. Both of them implement *foo()*, but only *B* implements
+*bar()* (*bar()* is not marked as abstract in *A*, because this is the default for interfaces if the function has no body).
 Now, if you derive a concrete class *C* from *A*, you, obviously, have to override *bar()* and provide an implementation.
```

Comment:
I suggest omitting the word 'obviously', because it may be not obvious to everyone